Source/UIKit/UITextField.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UITextField.swift
+//  Rex
+//
+//  Created by Rui Peres on 17/01/2016.
+//  Copyright © 2016 Neil Pankey. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+import UIKit
+
+extension UITextField {
+    
+    /// Sends the field's string value whenever it changes.
+    public var rex_textSignal: SignalProducer<String, NoError> {
+        return NSNotificationCenter.defaultCenter()
+            .rac_notifications(UITextFieldTextDidChangeNotification, object: self)
```

Comment:
I am testing it [here](https://github.com/RuiAAPeres/RACNest/), it seems fine, but on the other hand `rex_pressed` seems to be leaking: 
<img width="1210" alt="screen shot 2016-01-22 at 00 50 57" src="https://cloud.githubusercontent.com/assets/949286/12499407/3a2b0942-c0a2-11e5-9054-1b5c91b548f8.png">